project/SparkBuild.scala

```diff
@@ -643,6 +643,7 @@ object KubernetesIntegrationTests {
           "-t", imageTag.value,
           "-p", s"$bindingsDir/python/Dockerfile",
           "-R", s"$bindingsDir/R/Dockerfile",
+          "-b", s"java_image_tag=${sys.env.getOrElse("JAVA_IMAGE_TAG", "8-jre-slim")}",
```

Comment:
The default Java version is 8 if the K8s integration tests are launched by Maven so this PR complies with that.